lib/elastomer/client/errors.rb

```diff
@@ -109,7 +109,7 @@ def fatal
     IncompatibleVersionException = Class.new Error
 
     # Exception for client-detected invalid Elasticsearch parameter
```

Comment:
Do you mind updating this comment since it's now client-detected AND server response?

Reply:
not at all